analysis_file/Dockerfile

```diff
@@ -20,5 +20,5 @@ ADD . /app
 
 # USER is an environment variable which needs to be set when constructing this container e.g. via
 # docker run or docker container create. Use docker-run.sh to set these automatically.
-CMD pipenv run python analysis_file.py "$USER" /data/messages-input /data/survey-input.json \
+CMD pipenv run python -u analysis_file.py "$USER" /data/messages-input /data/survey-input.json \
```

Comment:
IIUC, this is forcing unbuffered stdio? Could we add this systematically to all the docker based Python files?